packages/opentelemetry-resources/src/platform/node/detectors/AwsEc2Detector.ts

```diff
@@ -30,8 +30,18 @@ class AwsEc2Detector implements Detector {
    * See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-identity-documents.html
    * for documentation about the AWS instance identity document endpoint.
    */
+  readonly HTTP_HEADER = 'http://';
+  readonly AWS_IDMS_ENDPOINT = '169.254.169.254';
+  readonly AWS_INSTANCE_TOKEN_DOCUMENT_PATH = '/latest/api/token';
+  readonly AWS_INSTANCE_IDENTITY_DOCUMENT_PATH =
+    '/latest/dynamic/instance-identity/document';
+  readonly AWS_INSTANCE_HOST_DOCUMENT_PATH = '/latest/meta-data/hostname';
   readonly AWS_INSTANCE_IDENTITY_DOCUMENT_URI =
     'http://169.254.169.254/latest/dynamic/instance-identity/document';
+  readonly AWS_INSTANCE_TOKEN_DOCUMENT_URI =
```

Comment:
Looks like unused.

Reply:
Yes, will delete this part, thank you for pointing out!